Makefile

```diff
@@ -153,7 +153,37 @@ check-bench-llvm:
 check-bench: check-bench-singlepass check-bench-llvm
 
 check: check-bench
-	cargo check --release --features backend-singlepass,backend-llvm,loader-kernel,debug
+	# TODO: We wanted `--workspace --exclude wasmer-runtime`, but for
```

Comment:
“but for”? Is the phrasing correct here?

Reply:
Doh, almost. "We would have used `--workspace --exclude wasmer-runtime`, but for <bug>" would have been correct. Fixed!